### DISCUSSION TITLE
Indents without spacing?

### DISCUSSION THREAD

**Discussion by DanieruSan2** [vkbo/novelWriter#2076] [Help]
I'm new to novelWriter and I'm trying to figure out how to properly indent. I learned that pressing Enter twice creates the indent, but it makes the spacing between paragraphs inconsistent with the actual line spacing. Is there any way to make the line spacing consistent with the creation of a new paragraph?

**Comment by vkbo** [comment]
Hmm, not sure I understand, An empty line creates a new paragraph. Indentation can be enabled in the manuscript tool for when you assemble your manuscript, but paragraph margins cannot be changed in the current version, so there is still a gap between them of half a line height.

In the next version it will be possible to change the values of heading and text margins.
